### PR TITLE
JavaScript for new framework

### PR DESCRIPTION
**Pull request by MalcolmVonMoJ**
- Adds in new GDS JavaScript.
- Adds new custom JS for searchbox show-hide.
- Related CSS and HTML changes needed for the GDS JS and for the search show-hide.

### CODE REVIEW

File: header.php [hunk 1]
```diff
@@ -25,7 +25,11 @@
 	flush();
 	?>
 </head>
-<body <?php body_class( 'js-enabled' ); ?>>
+<body <?php body_class(""); ?>>
+<script>
```
Comment (BNewing) [inline header.php:29]: What was the thinking behind putting it in a script tag vs in the JS file?

File: assets/js/hale-scripts.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+/*
+	hale script v1.0.0
+*/
+
+/*
+	Search button in header - show-hide the search field
+*/
+document.getElementById("search-show-hide").addEventListener('click',
```
Comment (BNewing) [inline assets/js/hale-scripts.js:8]: Do we need one for the hamburger menu too? (Sorry if that's been done in another PR!)

**Comment by BNewing** [comment]
Looks good! Left two questions but I suspect they're probably done in other branches so I'll take a look at other PRs now :)

File: assets/js/hale-scripts.js (hunk 2 above)
Comment (MalcolmVonMoJ) [reply assets/js/hale-scripts.js:8]: The menu one is dealt with by the GDS JavaScript already.

File: header.php (hunk 1 above)
Comment (MalcolmVonMoJ) [reply header.php:29]: This the the approach taken by GDS - I have not investigated why they choose this method.  However, you can see it is the way they tell people to do it [in their get started guide](https://frontend.design-system.service.gov.uk/get-started/#5-get-the-javascript-working).
